web_action_conditionable/static/src/js/views.js

```diff
@@ -1,22 +1,22 @@
-/*global openerp, _, $ */
-
-openerp.web_action_conditionable = function (instance) {
-  instance.web.View.include({
-    is_action_enabled: function(action) {
-      var attrs = this.fields_view.arch.attrs;
-      if (action in attrs) {
-	try {
-	  return this._super(action);
-	} catch(error) {
-	  var expr = attrs[action];
-	  var expression = py.parse(py.tokenize(expr));
-	  var cxt = this.dataset.get_context().__eval_context.__contexts[1];
```

Comment:
This data getter is needed to get the data. The fact that it is missing explains why the expr eval fails and why `state=='draft'` had to be changed to `'state'=='draft'` (which as @Yajo wrote above, cannot possibly work).